src/js/components/Box/__tests__/Box-style-test.tsx

```diff
@@ -198,6 +198,15 @@ describe('Box', () => {
     expect(container.firstChild).toMatchSnapshot();
   });
 
+  test('round with responive equal to false', () => {
```

Comment:
Does the snapshot effectively test the `responsive={false}` behavior or do we need to adjust the screen size in the jest test before taking the snapshot?

Reply:
we can make the screen smaller and then take a snap shot however if we did not have `responsive={false}` then snapshot would automatically add the 

```
@media only screen and (max-width:768px) {
  .c1 {
    border-radius: 24px;
  }
}
```

Reply:
Ah gotcha, I think the test is good as is then